### PR TITLE
Move arm controls to separate file

### PR DESCRIPTION
**Pull request by Tonnieboy300**
wait until we know if this actually works before merging please

### CODE REVIEW

**Comment by Tonnieboy300** [comment]
zachary why did you do that

**Comment by Tonnieboy300** [comment]
after testing it seems that the code works

**Comment by Tonnieboy300** [comment]
@Magician357 please review and merge

**Comment by Tonnieboy300** [comment]
wait

**Comment by Tonnieboy300** [comment]
nevermind I thought we had an issue but turns out the drivers didnt start the arm at the correct position

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Drivetrains/mecanumDriveRO_Arm.java
Comment (imetheones1) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Drivetrains/mecanumDriveRO_Arm.java:1]: Variable declarations should go after wait for start

**Comment by Tonnieboy300** [comment]
everything works merge please thanks

**Pull request merged**